docs/src/main/sphinx/admin/opentelemetry.md

```diff
@@ -43,6 +43,9 @@ is accessible from the coordinator and all workers of the cluster. The preceding
 example uses a observability platform deployment available by
 HTTP at the host `observe.example.com`, port `4317`.
 
+By default, Trino uses the gRPC protocol for exporting traces.
```

Comment:
Use the `tracing.exporter.protocol` property to configure the protocol for exporting traces. Defaults to the gRPC protocol with the `grpc` values. Set the value to `http/protobuf` for exporting traces using protocol buffers with HTTP transport.